src/deepmd_wrapper.F

```diff
@@ -151,6 +151,17 @@ SUBROUTINE DeepPotCompute(model, natom, coord, atype, cell, energy, force, viria
    SUBROUTINE deepmd_model_release(model)
       TYPE(deepmd_model_type)                            :: model
 
+      INTERFACE
+         SUBROUTINE DeleteDeepPot(dp) BIND(C, name="DP_DeleteDeepPot")
```

Comment:
Maybe use `model` instead of `dp` as argument name. It will be consistent and also avoid confusion with the `dp` that we import from `kinds.F`.